### ISSUE TITLE
Cruis'n Blast - 0100B41013C82000

### ISSUE THREAD

**Issue by mmarino4** [Ryujinx/Ryujinx-Games-List#3837]
#### Current on `master` : 1.1.145

The game is fully playable. The only obvious issue is on the vehicle select screen. The pictures on the bottom show up as black rectangles with borders. After a race they show up with glitchy green graphics in the borders. 

##### RAM: 16GB)

![ryujinx_capture_2022-02-18_08-42-23](https://user-images.githubusercontent.com/46875312/154698154-ec2fd7a7-fc8e-4d4e-ba60-b81657a7d4eb.png)
![ryujinx_capture_2022-02-18_08-48-11](https://user-images.githubusercontent.com/46875312/154698157-dd68d6ca-3943-4e05-8be4-b2415cd937e6.png)
![ryujinx_capture_2022-02-18_08-51-33](https://user-images.githubusercontent.com/46875312/154698145-ac63efdb-e157-49d9-9199-e16f20641c22.png)

**Comment by mmarino4** [comment]
The game crashes for me before getting to the main menu since 1.1.174

**Comment by markviv** [comment]
> The game crashes for me before getting to the main menu since 1.1.174

Does deleting your save file fix the problem?

See this issue:
https://github.com/Ryujinx/Ryujinx/issues/3330

**Comment by mmarino4** [comment]
> > The game crashes for me before getting to the main menu since 1.1.174
> 
> Does deleting your save file fix the problem?
> 
> See this issue: [Ryujinx/Ryujinx#3330](https://github.com/Ryujinx/Ryujinx/issues/3330)

Yes, it works fine if I delete the save files. It then creates it's own. If I try to replace the two files in folders 0 and 1 it stops working again.

**Comment by madthumbs** [comment]
Had the game crash before menu problem, but right clicking and purging the caches seems to fix it.

**Comment by scju** [comment]
>[...] right clicking and purging the caches seems to fix it. -Just a pain to do every time.

For me, it sometimes works but somtimes not. I have to do several times to get a chance.

**Comment by mrtnptrs** [comment]
Game Update Version : 1.07
Current on master : 1.1.816
Backend: Vulkan

The game is very much playable with good performance and no obvious problems or any crashes; tested by fully completing the first tournament/cup two times. There are some minor performance dips, but these also occur on real hardware. The black/glitchy vehicle selection images are a thing of the past. The game works great with resolution scaling. This game should thus be marked as "playable".

Hardware Specs :
CPU: i7 13700KF
GPU: Nvidia RTX 4090
RAM: 64 GB

![Ryujinx 1 1 816 - Cruis'n Blast v1 07 (0100B41013C82000) (64-bit) 22_05_2023 21_47_35](https://github.com/Ryujinx/Ryujinx-Games-List/assets/4948213/bd3eb3c4-9455-418f-995c-a30dbf8481ad)
![Ryujinx 1 1 816 - Cruis'n Blast v1 07 (0100B41013C82000) (64-bit) 22_05_2023 21_47_59](https://github.com/Ryujinx/Ryujinx-Games-List/assets/4948213/93cf11d4-a32f-4970-87ff-23a7f53132c4)
![Ryujinx 1 1 816 - Cruis'n Blast v1 07 (0100B41013C82000) (64-bit) 22_05_2023 21_50_22](https://github.com/Ryujinx/Ryujinx-Games-List/assets/4948213/facae1b8-3430-4c2c-ba18-4689120de9be)

[Ryujinx_1.1.816_2023-05-22_20-26-23.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/11536450/Ryujinx_1.1.816_2023-05-22_20-26-23.log)

**Comment by Nicktamer** [comment]
[Ryujinx_1.1.969_2023-07-28_11-31-31.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/12195401/Ryujinx_1.1.969_2023-07-28_11-31-31.log)
The game stuck on first loading screen. Sometimes it can reach the menu and run well but most of time it crash at loading screen.

**Comment by Nicktamer** [comment]
After many test the save is the problem. Sometimes it work but randomly the game corrupt the save. If the game corrupt the save you Need to delete the save on the folder "1" (not folder 0 if you want to keep progress in the game) Need to do this many times the game crash at startup

**Comment by Nicktamer** [comment]
[Ryujinx_1.1.969_2023-07-30_12-27-29.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/12208960/Ryujinx_1.1.969_2023-07-30_12-27-29.log)
At the beginning the game run fine but more you play more you have crash. And when finish game the save won't load and game crash at startup and the fix on the previous post no work
